pipeline/pipeline.py

```diff
@@ -120,7 +116,8 @@ def __contains__(self, item):
     DownloadUrlFile(control),
     WgetDownload(
         wpull_args,
-        accept_on_exit_code=AcceptAny(),
+        max_tries=1,
```

Comment:
`max_tries=1` might be too strict.  Are exit statuses 1, 2, and 3 recoverable in any way?  If so, the `--database` option combined with setting `keep_data` to `True` on the item should allow us to resume once the appropriate correction has been made to the pipeline.